examples/worlds/plot_3d.sdf

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" ?>
+<!--
+  Demo the Plot3D plugin.
+
+  Start this world and see how the pendulum's link's paths are plotted in the 3D scene as it moves.
+
+  The plugin is instantiated twice, once for each link of the pendulum.
+
+  Try out different parameters in each instance of the plugin to see how the plot changes.
+-->
+<sdf version="1.6">
+  <world name="shapes">
+
+    <gui fullscreen="0">
+
+      <!-- 3D scene -->
+      <plugin filename="GzScene3D" name="3D View">
+        <ignition-gui>
+          <title>3D View</title>
+          <property type="bool" key="showTitleBar">false</property>
+          <property type="string" key="state">docked</property>
+        </ignition-gui>
+
+        <engine>ogre2</engine>
+        <scene>scene</scene>
+        <ambient_light>0.4 0.4 0.4</ambient_light>
+        <background_color>0.8 0.8 0.8</background_color>
+        <camera_pose>5 -1.5 3 0 0.37 2.8</camera_pose>
+      </plugin>
+
+      <!-- World control -->
+      <plugin filename="WorldControl" name="World control">
+        <ignition-gui>
+          <title>World control</title>
+          <property type="bool" key="showTitleBar">false</property>
+          <property type="bool" key="resizable">false</property>
+          <property type="double" key="height">72</property>
+          <property type="double" key="width">121</property>
+          <property type="double" key="z">1</property>
+
+          <property type="string" key="state">floating</property>
+          <anchors target="3D View">
+            <line own="left" target="left"/>
+            <line own="bottom" target="bottom"/>
+          </anchors>
+        </ignition-gui>
+
+        <play_pause>true</play_pause>
+        <step>true</step>
+        <start_paused>true</start_paused>
+
```

Comment:
```suggestion
```

Reply:
Just curious, are you following any specific XML style for these suggestions? We haven't been enforcing any blank line policy so far.